### PR TITLE
Add a switch to control vm checking for gating cases

### PR DESCRIPTION
**Pull request by xiaodwan**
Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

**Comment by xiaodwan** [comment]
@kylazhang @chloerh please help to review this pr, and merge it if it has no problem.

File: v2v/tests/cfg/convert_from_file.cfg [hunk 1]
```diff
@@ -138,6 +138,9 @@
                     input_mode = "vmx"
                     only source_esx
                     no output_mode.local
+                    # For gating, skipping vm check in openstack because of IP
+                    # Address problem
+                    skip_vm_check = SKIP_VM_CHECK_OPENSTACK
```
Comment (sathnaga) [inline v2v/tests/cfg/convert_from_file.cfg]: `SKIP_VM_CHECK_OPENSTACK` where is this defined?, if this something internal env variable, avoid using it.

you can use it in your local test config and run this test, no need for this change.

Comment (xiaodwan) [reply v2v/tests/cfg/convert_from_file.cfg]: This is a MARCO which will be replaced with the real value based on the real job or the needs how you run the script. The value is define in our libvirt-ci project. You can also find other similar MARCOs in other cfg files in tp-libvirt.

Comment (sathnaga) [reply v2v/tests/cfg/convert_from_file.cfg]: I guess libvirt-ci is internal, but not tp-libvirt, and it is always good practice, not to have internal code here.

Comment (sathnaga) [reply v2v/tests/cfg/convert_from_file.cfg]: am fine to have `skip_vm_check` as a new param, just that keep it a known default value and override with your needs in your `libvirt-ci` project.

Comment (balamuruhans) [reply v2v/tests/cfg/convert_from_file.cfg]: Agree with @sathnaga, @xiaodwan you can always override the param `skip_vm_check` --vt-extra-params or from your test config in the CI setup. this is not right way to do in upstream.

Comment (xiaodwan) [reply v2v/tests/cfg/convert_from_file.cfg]: @sathnaga @balamuruhans Thanks for your suggestion. I updated the code to set default value for skip_vm_check explicitly. Please help to check again.

**Pull request merged**